translator/trace/jaeger/jaegerproto_to_protospan.go

```diff
@@ -203,6 +204,14 @@ func jProtoTagsToAttributes(tags []model.KeyValue) (string, tracepb.Span_SpanKin
 				sKind = tracepb.Span_SERVER
 			}
 
+		case tracetranslator.TagError:
+			if statusCodePtr == nil {
+				// Only set this if tracetranslator.TagStatusCode does not exists.
+				// The tracetranslator.TagStatusCode branch will overwrite the value if exists.
```

Comment:
So technically it is possible to have in the input "error=true" and "status=OK" we will end up ignoring the "error" and will output "status=OK". I think this is OK, I don't see a better way to handle this conflicting input tags.

Reply:
Correct, I am not trying to fix wrong data.